app/models/presenter/auction.rb

```diff
@@ -32,6 +32,7 @@ def initialize(auction)
       :to_param,
       :type,
       :updated_at,
+      :lowest_bid,
```

Comment:
Lowest bid is used to calculate winning bid. If we were using a plain Ruby object in the RulesFactory (like we do in its first two uses) we don't have to mess with the view model but since the API uses the controller and that returns a view object for auction, we need to delegate it down. We def need to reconsider how heavily we are relying on the auction view model /presenter because right now they are all tangled up!